### PR TITLE
fix windows compat checks, compat check also BEFORE merging to master

### PR DESCRIPTION
**Pull request by exaexa**

### CODE REVIEW

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 4963ed6e is formatted properly.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@ variables:
 .global_trigger_compat_tests: &global_trigger_compat_tests
   rules:
     - if: $CI_COMMIT_BRANCH == "master"
+    - if: $CI_EXTERNAL_PULL_REQUEST_TARGET_BRANCH_NAME == "master"
```
Comment (laurentheirendt) [inline .gitlab-ci.yml:41]: maybe the test should be on the source branch being `develop`, no?

Comment (exaexa) [reply .gitlab-ci.yml:41]: I wanted to be a bit more explicit in saying "anything that gets to master needs to pass here", which may be stuff from other branches than develop.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit d5cd6f03 is formatted properly.

**Review by laurentheirendt — APPROVED** [review]

**Pull request merged**